### PR TITLE
remove pcidevices chart dependency

### PR DESCRIPTION
**Pull request by ibrokethecloud**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**

PR to move PCIDevices to an addon. 
This involves removing the pcidevices subchart and also adding the upgrade addon manifest.

**Solution:**

**Related Issue:**
https://github.com/harvester/harvester/issues/2913
**Test plan:**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -65,48 +65,56 @@ require (
...
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
+	github.com/StackExchange/wmi v1.2.1 // indirect
+	github.com/go-ole/go-ole v1.2.6 // indirect
+	github.com/jaypipes/ghw v0.9.0 // indirect
+	github.com/jaypipes/pcidb v1.0.0 // indirect
+	github.com/mitchellh/go-homedir v1.1.0 // indirect
+)
```
Comment (guangbochen) [inline go.mod]: is this a go mod behavior or can this be moved to the below section?

File: pkg/data/crd.go [hunk 2]
```diff
@@ -31,6 +32,19 @@ func createCRDs(ctx context.Context, restConfig *rest.Config) error {
...
+				WithColumn("Device Id", ".status.deviceId").
+				WithColumn("Node Name", ".status.nodeName").
+				WithColumn("Description", ".status.description").
+				WithColumn("Kernel Driver In Use", ".status.kernelDriverInUse"),
+			crd.NonNamespacedFromGV(devices.SchemeGroupVersion, "PCIDeviceClaim", devices.PCIDeviceClaim{}).
+				WithColumn("Address", ".spec.address").
+				WithColumn("Node Name", ".spec.nodeName").
+				WithColumn("User Name", ".spec.userName").
+				WithColumn("Kernel Driver Το Unbind", ".status.kernelDriverToUnbind").
+				WithColumn("Passthrough Enabled", ".status.passthroughEnabled"),
```
Comment (guangbochen) [inline pkg/data/crd.go]: ~~we can leave the column empty if the pci-device controller will overwrite the CRD eventually so we don't need to manage the update in the future.~~

**Review by guangbochen — APPROVED** [review]
LGTM

**Review by bk201 — APPROVED** [review]
Thanks!

**Pull request merged**